### ISSUE TITLE
where can i get the 64-sphere's sovler file

### ISSUE THREAD

**Issue by hrlqq** [wy1iu/sphereface#74]
i want to train it

**Comment by wy1iu** [comment]
The solver file is basically the same as the Sphereface-20.

**Comment by hrlqq** [comment]
what the finial loss is normal?

**Comment by zuoqing1988** [comment]
@hrlqq I have tried many times, and it's hard to converge.
First, I find batch_size should be set to 96. (128 or 256 will result in failure on GTX1080Ti).
Second, initial learning rate should be 0.01 rather than 0.1.
Third, using 'SINGLE' can converge easily, but finetuning ‘QUADRUPLE’ with lr = 1e-4 often diverge.

**Comment by zuoqing1988** [comment]
@hrlqq When lambda_min = 10, the loss will be less than 1.0 after convergence.
When lambda_min = 5, the loss will be less than 2.
When lambda_min = 2, the loss will be less than 5.
When lambda_min = 1, the loss wil be less than 10.

**Comment by hrlqq** [comment]
@zuoqing1988  cause my  GPU's memery isnot that big, i change the batch size to 128, and i change the lr to  0.05,  and enlarge setpvalue max_iter ,the final loss is from 9.x to 2.x or 1.x

**Comment by zuoqing1988** [comment]
@hrlqq 
I have tested several subsets of ms_celeb_1m.  Until now, only one experiment converges well with about 30,000 classes and total 2.7 million images.  
I set batch_size =96 for all 64-layer experiments.
Firstly, I trained using 'SINGLE', with solver file lr = 0.01, stepvalue = 64,000, 72,000, 96,000, max_iter = 120,000, and model file base = 1000, gamma = 0.02, lambda_min = 50. The acc is around 98% on LFW.
Secondly, I finetuned from the `SINGLE' result with solver file lr=0.01, stepvalue = 160,000, 240,000, max_iter = 280,000, and model file base = 10, gamma = 0.01, lambda_min = 10. The acc is around 99% on LFW.
Thirdly, I changed lambda_min = 5 and set stepvalue = 160,000,  240,000, 400,000, max_iter = 560,000, and trained with argument "-snapshot xxx_iter_280000.caffemodel". The acc is 99.25% on LFW.
I'm not satisfied with this accuracy, as I have obtained 99.35% with the 20-layer model.
At this moment, I'm continuely running from iter = 560,000 with lambda_min = 1 and lambda_min = 2, respectively.

**Comment by haoranD** [comment]
in caffe-sphereface , i change the cmakefile.txt and makefileconfig, and successfully make all,but making test was failed,so can't finish the installation.but my own caffe is ok

help！please

src/caffe/test/test_neuron_layer.cpp:17:38: fatal error: caffe/layers/log_layer.hpp: can't find
compilation terminated.
Makefile:581: recipe for target '.build_release/src/caffe/test/test_neuron_layer.o' failed
make: *** [.build_release/src/caffe/test/test_neuron_layer.o] Error 1
make: ***
